examples/creatives/base_simid_creative.js

```diff
@@ -20,7 +20,17 @@ class BaseSimidCreative {
      * The most recent video state from the player.
      * @protected {?Object}
      */
-    this.videoState = {};
+    this.videoState = {
+      currentSrc:'',
+      currentTime: -1, // Time not yet known
+      duration: -1, // duration unknown
+      ended: false,
+      muted: false,
+      paused: false,
+      volume: 0.5,
+      fullscreen: false //TODO add this to environment data in spec
```

Comment:
I think you can remove the todo now, looks like it is in the spec

Reply:
Done.